org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/ZkReportViewer.java

```diff
@@ -901,90 +901,92 @@ private void renderReport() {
 	}
 	
 	private void onPreviewReport() {
-		try {

```

Comment:
to limit the amount of diff, I think better just add a `if (media == null) return;` check at top of the method.

Reply:
it's ok

Reply:
Ajusted